GitUp/Application/AppDelegate.m

```diff
@@ -68,6 +79,113 @@ - (BOOL)canBecomeKeyWindow {
 
 @end
 
+@interface GIWelcomeWindow__DraggingInfoExtractor : NSObject
+- (BOOL)canDragItem:(id<NSDraggingInfo>)sender;
+- (NSArray <NSURL *>*)draggingItems:(id<NSDraggingInfo>)sender;
+@end
+
+@implementation GIWelcomeWindow__DraggingInfoExtractor
+- (BOOL)canDragItem:(id<NSDraggingInfo>)sender {
+  return [self draggingItems:sender].count > 0;
+}
+- (NSArray <NSURL *>*)draggingItems:(id<NSDraggingInfo>)sender {
+  return [[sender draggingPasteboard] readObjectsForClasses:@[NSURL.class] options:nil];
+}
+@end
+
+@interface GIWelcomeWindow__DestinationView : NSView
```

Comment:
Let's not underscore our class names. Just not a convention within GitUp or Obj-C at large.